core/dbt/config/project.py

```diff
@@ -59,6 +59,17 @@
 --no-version-check
 """
 
+MALFORMED_PACKAGE_ERROR = """\
+The packages.yml file in this project is malformed. Please double check the contents
```

Comment:
I see that our style guide checker (flake8) failed with this error:

> core/dbt/config/project.py:63:80: E501 line too long (84 > 79 characters)

Can you just split this onto two lines to appease flake8? Thanks!